### PR TITLE
Update frontmatter.md

### PR DESCRIPTION
**Pull request by i-give-up**
Mention that you can set the front-matter variable `layout` to `null` if you don't want to use any layout file.

### CODE REVIEW

**Comment by DirtyF** [comment]
@i-give-up Thanks for the add, FWIW since 3.5.0 you can also use `layout: none` with a slightly difference:

> Using `layout: none` will now produce a file with no layout. Equivalent to `layout: null`, with the exception that `none` is a truthy value and won't be overwritten by front matter defaults.

Maybe we could also mention this?

File: docs/_docs/frontmatter.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ front matter of a page or post.
 
           If set, this specifies the layout file to use. Use the layout file
           name without the file extension. Layout files must be placed in the
-          <code>_layouts</code> directory.
+          <code>_layouts</code> directory. Set this to <code>null</code> if you
+          don't want to use any layout file.
```
Comment (ashmaroli) [inline docs/_docs/frontmatter.md]: The downside to the proposition is that `layout: null` will be overridden by `frontmatter_defaults` defined in for the document (?) in the config file. As of Jekyll 3.5.x, this is overcome by setting `layout: none` in the document. (Works only for `Jekyll:Document` objects. For regular `Jekyll::Page` objects (like `./about.md`), there's a proposed port via an open PR.)

Comment (i-give-up) [reply docs/_docs/frontmatter.md]: So, currently to prevent override by `defaults` in `_config.yml`, you have to use?

- `layout: none` for documents
- `layout: null` for pages

Seems a bit messy.

Comment (ashmaroli) [reply docs/_docs/frontmatter.md]: nope.. setting `layout: none` means it will not be overridden by `_config.yml`

Comment (i-give-up) [reply docs/_docs/frontmatter.md]: I know. That's why I said currently you have to use `layout: null` for pages. If you use `layout: none` for pages it'll try to use a layout named `none`.

So to summarize

- Using `layout: null` will produce a file without layout file, but this will be overridden if the file is a Document and has a layout defined in front matter defaults.
- (version 3.5.0 and above) Using `layout: none` in a Document produce a file without a layout file. If used in a Page, it will try to use a layout named `none`.

Comment (ashmaroli) [reply docs/_docs/frontmatter.md]: yes, that is 99.99% correct.. :smiley:  If things go as I hope, `v3.6.0` might land with `layout: none` applicable to Pages as well..

**Comment by i-give-up** [comment]
Ok, made some changes. It's not as pretty as before, but should be a bit more accurate in describing `layout: null` and `layout: none`.

File: docs/_docs/frontmatter.md [hunk 2]
```diff
@@ -65,9 +65,23 @@ front matter of a page or post.
 
           If set, this specifies the layout file to use. Use the layout file
           name without the file extension. Layout files must be placed in the
-          <code>_layouts</code> directory.
+          <code>_layouts</code> directory. 
 
         </p>
+        <ul>
```
Comment (DirtyF) [inline docs/_docs/frontmatter.md]: Note: `tbody td ul ` font-size is currently not the same as `tbody td p` elements.  This should be taken care of in another PR. (or just limit ourselves to `p` tags in tables.)

**Comment by DirtyF** [comment]
@jekylldocs: merge +docs
